### ISSUE TITLE
Some lidar timestamps are not synchronized to seven ring cameras.

### ISSUE THREAD

**Issue by synsin0** [argoverse/av2-api#112]
I find that when using the SensorDataloader, some lidar timestamps are not synchronized to seven ring cameras. Some only give six synchronized imageries. Is the tolerance of synchronization too strict?

**Comment by chensiweiTHU** [comment]
Yes, when I tried to find the closest images from the point cloud samples, the longest ineterval between an image and its closest point cloud is 147 ms. It seems that some of the images are lost
